ckan/templates/user/edit_user_form.html

```diff
@@ -24,7 +24,7 @@
 
   <fieldset>
     <legend>{{ _('Change password') }}</legend>
-    {{ form.input('old_password', type='password', label=_('Old Password'), id='field-password', value=data.oldpassword, error=errors.oldpassword, classes=['control-medium'], attrs={'autocomplete': 'off'} ) }}
+    {{ form.input('old_password', type='password', label=_('Sysadmin Password') if c.is_sysadmin else _('Old Password'), id='field-password', value=data.oldpassword, error=errors.oldpassword, classes=['control-medium'], attrs={'autocomplete': 'off'} ) }}
```

Comment:
this line is getting really long, I would split it up (jinja2 allows newlines between tokens in {{ }} blocks)